src/hardware/voodoo.cpp

```diff
@@ -188,8 +181,9 @@ void VOODOO_Destroy(Section* sec) {
 }
 
 void VOODOO_Init(Section* sec) {
+	constexpr uint32_t initial_lfb_masked = VOODOO_INITIAL_LFB & 0xffff0000;
     if (!voodoo_dev) {
-        voodoo_current_lfb = (VOODOO_INITIAL_LFB & 0xffff0000);
+        voodoo_current_lfb = initial_lfb_masked;
         voodoo_dev = std::make_unique<VOODOO>(sec);
```

Comment:
At the top of the file, I see:

``` c++
constexpr uint32_t voodoo_lfb_mask = 0xffff0000;
static uint32_t voodoo_current_lfb = (VOODOO_INITIAL_LFB & voodoo_lfb_mask);
```

Which is very similar to what's happening here (ie: `voodoo_lfb_mask` is the same as this `0xffff0000` variable).  Maybe `initial_lfb_masked` could be lifted to the top of the file too, and it be used to set `voodoo_current_lfb`?

Reply:
I agree, and I have since fixed it on my end. (If you see the commits, I had it like your suggestion before, I believe)